packages/style/scss/elements/btn.scss

```diff
@@ -249,6 +249,50 @@
     &.collapsible-header {
         border-radius: 0;
     }
+
+    &.mod-loading {
+        &.mod-danger .loading-spinner,
+        &.mod-link .loading-spinner,
+        &.mod-confirm .loading-spinner,
+        &.mod-confirm-error .loading-spinner,
+        &.mod-confirm-success .loading-spinner {
+            display: none;
+        }
+
+        &:disabled .loading-spinner:before,
+        &.state-disabled .loading-spinner:before {
+            background-color: var(--white);
```

Comment:
Would using a mixins or a function to change the colors of the spinner be more appropriate? 🤔 

Because I feel like changing the color of the bg and border in the :before could be needed in other cases and right now the component in not super flexible 🤔